test/cmd/create.sh

```diff
@@ -72,18 +72,6 @@ run_kubectl_create_error_tests() {
   # Passing no arguments to create is an error
   ! kubectl create || exit 1
 
-  ## kubectl create should not panic on empty string lists in a template
-  ERROR_FILE="${KUBE_TEMP}/validation-error"
-  kubectl create -f hack/testdata/invalid-rc-with-empty-args.yaml "${kube_flags[@]}" 2> "${ERROR_FILE}" || true
-  # Post-condition: should get an error reporting the empty string
-  if grep -q "unknown object type \"nil\" in ReplicationController" "${ERROR_FILE}"; then
```

Comment:
this was a false positive where client-side validation failed something server-side validation would understand and persist. removing it is correct.